App.js

```diff
@@ -16,72 +16,61 @@ import { ParentTips, ScenerioTips } from './pages/ParentTips.js';
 import {
     StyledView,
     StyledHomeButtonView,
-    StyledHomeText
+    StyledLogo
 } from './components/StyledView';
 import { Play } from './pages/Play.js';
 import { HowToPlay } from './pages/HowToPlay.js';
 
 function HomeScreen({ navigation }) {
     return (
         <StyledView>
-            <Image
-                source={require('./assets/genconnect_logo-black.png')}
-                style={{
-                    width: 303,
-                    height: 100,
-                    marginTop: 170
-                }}
-            />
-            <StyledHomeButtonView>
-                <StyledHomeText
-                    style={{ zIndex: 1, left: 10, top: 45 }}
-                    onPress={() => navigation.navigate('Play')}
-                >
-                    {' '}
-                    PLAY{' '}
-                </StyledHomeText>
-
-                <StyledHomeText
-                    style={{ zIndex: 1, right: 18, top: 35, color: '#EE3282' }}
-                    onPress={() => navigation.navigate('How To Play')}
-                >
-                    {' '}
-                    HOW TO PLAY{' '}
-                </StyledHomeText>
-
-                <StyledHomeText
-                    style={{ zIndex: 1, left: 13, top: 250 }}
-                    onPress={() => navigation.navigate('Parent Guide')}
-                >
-                    {' '}
-                    PARENT GUIDE{' '}
-                </StyledHomeText>
+            <ScrollView>
+                <StyledLogo>
+                    <Image
+                        source={require('./assets/genconnect_logo-black.png')}
+                        style={{
+                            width: 350,
+                            height: 130,
+                            marginLeft: 10,
+                            marginRight: 10
+                        }}
+                    />
+                    <Text
+                        style={{
+                            fontWeight: '800',
+                            fontFamily: 'Avenir',
+                            fontSize: 18
+                        }}
+                    >
+                        {' '}
+                        The game that gets people talking!{' '}
```

Comment:
Should make a separate styled component for this in case we want to reuse this text in the future.

Reply:
Manually adding space like `{' '}` is not ideal. Should find a way to position it with css.

Reply:
Okay